website/content/docs/job-specification/group.mdx

```diff
@@ -81,19 +81,24 @@ job "docs" {
   own [`shutdown_delay`](/docs/job-specification/task#shutdown_delay)
   which waits between deregistering task services and stopping the task.
 
-- `stop_after_client_disconnect` `(string: "")` - Specifies a duration
-  after which a Nomad client that cannot communicate with the servers
-  will stop allocations based on this task group. By default, a client
-  will not stop an allocation until explicitly told to by a server. A
-  client that fails to heartbeat to a server within the
-  [`heartbeat_grace`] window and any allocations running on it will be
-  marked "lost" and Nomad will schedule replacement
-  allocations. However, these replaced allocations will continue to
-  run on the non-responsive client; an operator may desire that these
-  replaced allocations are also stopped in this case — for example,
-  allocations requiring exclusive access to an external resource. When
-  specified, the Nomad client will stop them after this duration. The
-  Nomad client process must be running for this to occur.
+- `stop_after_client_disconnect` `(string: "")` - Specifies a duration after
+  which a Nomad client will stop allocations, if it cannot communicate with the
+  servers. By default, a client will not stop an allocation until explicitly
+  told to by a server. A client that fails to heartbeat to a server within the
+  [`heartbeat_grace`] window and any allocations running on it will be marked
+  "lost" and Nomad will schedule replacement allocations. However, these
+  replaced allocations will continue to run on the non-responsive client; an
+  operator may desire that these replaced allocations are also stopped in this
+  case — for example, allocations requiring exclusive access to an external
+  resource. When specified, the Nomad client will stop them after this duration.
```

Comment:
This sentence has run away on us a bit and has a bunch of passive voice and third-person reference to the operator. Maybe break it up into:

```suggestion
  "lost" and Nomad will schedule replacement allocations. The replaced 
  allocations will normally continue to run on the non-responsive client. But
  you may want them to stop instead — for example, allocations requiring 
  exclusive access to an external resource. When specified, the Nomad client 
  will stop them after this duration.
```